lib/src/zap/zap_priv.h

```diff
@@ -246,6 +247,18 @@ struct zap {
 
 	/** Pointer to the transport's private data */
 	void *private;
+
+	/**
+	 * \brief Send data to peer using map.
+	 *
+	 * No data is copied out from the described memory region. When the
+	 * operation is completed, \c ZAP_EVENT_SEND_COMPLETE shall be
+	 * delivered. If the operation synchronously failed, \c
+	 * ZAP_EVENT_SEND_COMPLETE must not be delivered. The \c context is the
```

Comment:
This is a man page, not a spec. So "must not be" is confusing because it is might be interpreted by the reader as a something there application must ensure as opposed to something the API guarantees.

Reply:
I changed this to "will not be".